### PR TITLE
Choose fonts feature

### PR DESCRIPTION
**Pull request by 20chix**

### CODE REVIEW

**Comment by 20chix** [comment]
@benji011  Would you like to review this feature?

File: README.md [hunk 1]
```diff
@@ -65,13 +65,16 @@ Contributions are always welcome, feel free to submit a PR, I'll review it and i
...
+- [x] Allow user to change their username after they sign-up
 - [ ] Add some tests
 - [ ] Add QR Code feature
-- [ ] Allow user to choose a background color
-- [ ] For populare social media add a 'Quick add section' where user just enter the username instead of the whole link
-- [x] When user upload profile image compress it, so it doesn't take much space in the server
-- [ ] Allow user to change their username after they sign-up
 - [ ] Add caching feature, instead pulling all the data every time.
+- [ ] Add Stats in a separate tab, example how many views and links clicks the profile had.
+- [ ] Allow user to choose from different templates when diplaying their profiles.
```
Comment (benji011) [inline README.md]: just a small typo

```suggestion
- [ ] Allow user to choose from different templates when displaying their profiles. 
```

**Comment by benji011** [comment]
Besides just 1 comment, everything else looks good to me!

**Review by benji011 — APPROVED** [review]

**Comment by 20chix** [comment]
Thanks @benji011

**Pull request merged**